### DISCUSSION TITLE
搜索时offset + limit不能超过16384

### DISCUSSION THREAD

**Discussion by lymbjtungirc** [milvus-io/milvus#26517] [Q&A and General discussion] [answered]
你们好(*^▽^*)！  想问下Milvus的这个搜索限制，是我在每个partition下存储超过16384条数据的话，超过的数据量是查不出来的对嘛。比如我有遍历所有数据，进行改查的需求。

**Comment by yhmo** [comment, marked as answer]
可以根据条件把数据一段段取出。比如通过query(expr="id>=0 && id < 100"), query(expr="id>=100 && id < 200"), query(expr="id>=200 && id < 300")...... 这样多次取出的方式来拿。

**Reply by yhmo** [reply]
PS: 尽量用英文问答

**Reply by xiaofan-luan** [reply]
We support search iterator in 2.3 and hopefully that could solve the problem

**Reply by lymbjtungirc** [reply]
ok，thx
